generate.py

```diff
@@ -0,0 +1,67 @@
+from collections import OrderedDict
```

Comment:
I'd like this in a separate repo. I can create one under the ST organization GitHub

Reply:
Do whatever you want!
bt this file is based on https://github.com/SillyTavern/SillyTavern/pull/2098/files#diff-3853ac2189202f20185621d539c6b57da80fa0a15c35c5bf239577b4c8385032 . You may want to indicate its author.

Reply:
You can continue working on this and submit it here: https://github.com/SillyTavern/SillyTavern-i18n